### PR TITLE
Improve speech commands dataset preparation script

### PR DESCRIPTION
**Pull request by titu1994**
# Changelog
- Adds more logs to track progress
- Adds a flag `--skip_duration` which is useful to speed up data preparation for colab

These changes are required before notebook for speech commands can be submitted

### CODE REVIEW

File: scripts/process_speech_commands_data.py [hunk 1]
```diff
@@ -314,17 +339,30 @@ def __process_data(data_folder: str, dst_folder: str, rebalance: bool = False, c
...
 def main():
     parser = argparse.ArgumentParser(description='Google Speech Command Data download')
     parser.add_argument("--data_root", required=True, default=None, type=str)
     parser.add_argument('--data_version', required=True, default=1, type=int, choices=[1, 2])
     parser.add_argument('--class_split', required=False, default='all', type=str, choices=['all', 'sub'])
     parser.add_argument('--rebalance', required=False, action='store_true')
+    parser.add_argument('--skip_duration', required=False, action='store_true')
     parser.add_argument('--log', required=False, action='store_true')
-    parser.set_defaults(log=False, rebalance=False)
+    parser.set_defaults(log=False, rebalance=False, skip_duration=False)
```
Comment (blisc) [inline scripts/process_speech_commands_data.py:365]: Isn't this line redundant since they are "store_true" anyways.

Comment (titu1994) [reply scripts/process_speech_commands_data.py:365]: That seems to be the way argparse doc suggests using these action flags. It avoids a case where the flag is none cause no value is set and no flag is passed.

Comment (blisc) [reply scripts/process_speech_commands_data.py:365]: Do you have the link to where this is recommended? I'm never heard of it done this way. Would be nice to know if this is the recommended way.

Comment (titu1994) [reply scripts/process_speech_commands_data.py:365]: So trying to find that doc, it seems I need to brush up on py3.x argparse. The above behaviour is in 2.7, and 3.2+ auto sets the default as the inverse of store_*. We can remove it safely. It's still used in various code bases where the flag itself is unclear as to what the default value is. So I think it's good to have the defaults explicitly set, even if not necessary anymore. 
Source: https://docs.python.org/3/library/argparse.html#action

**Pull request merged**